admin/metabox/class-metabox-collapsibles-section.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays a metabox tab that consists of collapsible sections.
+ */
+class WPSEO_Metabox_Collapsibles_Sections extends WPSEO_Abstract_Metabox_Tab_With_Sections {
+
+	/**
+	 * @var WPSEO_Metabox_Collapsible[]
+	 */
+	private $collapsibles = array();
+
+	/**
+	 * Constructor.
+	 *
+	 * @param string $name         The name of the section, used as an identifier in the html.
+	 *                             Can only contain URL safe characters.
+	 * @param string $link_content The text content of the section link.
+	 * @param array  $collapsibles The metabox collapsibles (`WPSEO_Metabox_Collapsible[]`) to be included in the section.
+	 * @param array  $options      Optional link attributes.
+	 */
+	public function __construct( $name, $link_content, array $collapsibles = array(), array $options = array() ) {
+		parent::__construct( $name, $link_content, $options );
+
+		$this->collapsibles = $collapsibles;
+	}
+
+	/**
+	 * Outputs the section content if any tab has been added.
+	 */
+	public function display_content() {
+		if ( $this->has_sections() ) {
+			printf( '<div id="%1$s" class="wpseo-meta-section">', esc_attr( 'wpseo-meta-section-' . $this->name ) );
+			echo '<div class="wpseo_content_wrapper">';
```

Comment:
We shouldn't use the `wpseo_content_wrapper` CSS class within the metabox: this class uses `display: table` only because it's meant to be used in the settings pages, because of the banners sidebar.